deploy/examples/cluster-stretched.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     stretchCluster:
       # The ceph failure domain will be extracted from the label, which by default is the zone. The nodes running OSDs must have
       # this label in order for the OSDs to be configured in the correct topology. For topology labels, see
-      # https://rook.github.io/docs/rook/latest/ceph-cluster-crd.html#osd-topology.
+      # https://rook.io/docs/rook/latest/ceph-cluster-crd.html#osd-topology.
```

Comment:
This link doesn't work. Please also check other links, I just got lucky on this one.

Reply:
actually, the broken links were for the place where I was only changing removing `github.io` should be good(for almost all the places 😅 )